common/redshift/Makefile

```diff
@@ -93,6 +93,9 @@ storage-upload: build
 	for f in $(notdir $(wildcard dist/*.zip)); do \
 		STATEMENT_SQL="CREATE OR REPLACE LIBRARY $(RS_SCHEMA) LANGUAGE plpythonu FROM '$(AWS_S3_BUCKET)$(notdir $$f)' CREDENTIALS 'aws_access_key_id=$(AWS_ACCESS_KEY_ID);aws_secret_access_key=$(AWS_SECRET_ACCESS_KEY)';"; $(AWS_RUN_STATEMENT_SQL); \
 	done
+ifneq ("$(wildcard python_requirements.txt)","")
+	. $(COMMON_DIR)/redshift/libraryinstaller.sh -f python_requirements.txt
```

Comment:
Change `.` by `bash` because it's a bash script. In my case, it's not working with `.`